### ISSUE TITLE
NetBSD support (for xhovacui)

### ISSUE THREAD

**Issue by esad** [sgerwk/pdftoroff#1]
Hi, 

thanks for you great work, hovacui works wonderfully on small screens and low-end devices.

I got xhovacui to compile (and it seems to run fine) on NetBSD (which doesn't have `<linux/fb.h>`), by removing the fb dependency and adding few headers:

```diff
diff --git a/Makefile b/Makefile
index 1202e05..9215db4 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ pdftoroff pdffit pdfrects pdfrecur: pdfrects.o
 hovacui fbhovacui drmhovacui xhovacui: pdfrects.o
 fbhovacui: cairofb.o vt.o cairoio-fb.o cairoui.o hovacui.o fbhovacui.o
 drmhovacui: cairodrm.o vt.o cairoio-drm.o cairoui.o hovacui.o drmhovacui.o
-xhovacui: cairofb.o vt.o cairoio-x11.o cairoui.o hovacui.o xhovacui.o
+xhovacui: cairoio-x11.o cairoui.o hovacui.o xhovacui.o
 hovacui: cairofb.o cairodrm.o vt.o cairoio-x11.o cairoio-fb.o cairoio-drm.o \
 cairoui.o hovacui.o hovacui-main.o
 cairoui: cairofb.o vt.o cairoio-fb.o cairoio-x11.o cairoui.o cairoui-main.o
diff --git a/cairoio-x11.c b/cairoio-x11.c
index aa2afec..ac91ac8 100644
--- a/cairoio-x11.c
+++ b/cairoio-x11.c
@@ -1,6 +1,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <string.h>
+#include <sys/time.h>
 #include <X11/Xlib.h>
 #include <X11/keysym.h>
 #include <X11/Xatom.h>
@@ -8,7 +9,6 @@
 #include <cairo.h>
 #include <cairo-xlib.h>
 #include "cairoio.h"
-
 /*
  * name of the program, as used in the window title
  */
diff --git a/cairoui.c b/cairoui.c
index 33b6766..9407cf6 100644
--- a/cairoui.c
+++ b/cairoui.c
@@ -260,6 +260,7 @@
 #include <string.h>
 #include <signal.h>
 #include <time.h>
+#include <stdarg.h>
 #include <ctype.h>
 #include "cairoio.h"
 #include "cairoui.h"
```

Was there a reason to link against vt.o and cairofb.o for the xhovacui target?

**Comment by sgerwk** [comment]
Somehow, I didn't notice this issue until recently. You are right on all
counts: these dependencies are unnecessary, the includes are.

Thanks!

Fixed in 9b727c9dff70e83a19394b427077df70e67e71fc

**Comment by sgerwk** [comment]
Somehow, I didn't notice this issue until recently. You are right on all
counts: these dependencies are unnecessary, the includes are.

Thanks!

Fixed in 9b727c9dff70e83a19394b427077df70e67e71fc
